ikasaneip/rest/rest-module-client/src/test/java/org/ikasan/rest/client/TestConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.ikasan.rest.client;
```

Comment:
What is this class actually doing?

Reply:
This Class is used by ConfigurationRestServiceImplTest in order to load configuration related beans to spring context. that is required to fully TEST Configuration REST endpoint

```
@ContextConfiguration(classes = { TestConfiguration.class }) 
```